### PR TITLE
pdn: Fix flags documentation.

### PR DESCRIPTION
**Pull request by titan73**

### CODE REVIEW

**Comment by titan73** [comment]
Actually -verbose flag of pdngen is deprecated.

File: src/pdn/README.md [hunk 1]
```diff
@@ -168,7 +167,7 @@ define_power_switch_cell
     -ground ground_pin
     -name name
     -power unswitched_power_pin
-    -power_switchable power_switchable_pin
```
Comment (gadfort) [inline src/pdn/README.md:171]: the tcl code has this as `power_switchable`

**Review by gadfort — CHANGES_REQUESTED** [review]

File: src/pdn/README.md (hunk 1 above)
Comment (titan73) [reply src/pdn/README.md]: Fixed. Thanks.

**Review by gadfort — APPROVED** [review]
@titan73 try not to force-push after opening a PR it makes reviewing requested changed harder.

**Comment by titan73** [comment]
> @titan73 try not to force-push after opening a PR it makes reviewing requested changed harder.

@gadfort What should I do? A commit for each update?

**Comment by gadfort** [comment]
> > @titan73 try not to force-push after opening a PR it makes reviewing requested changed harder.
> 
> @gadfort What should I do? A commit for each update?

yeah, that preserves the diff history

**Comment by titan73** [comment]
> yeah, that preserves the diff history
But this will pollute master history once PR is merged, right?

**Comment by titan73** [comment]
> 
> yeah, that preserves the diff history

But this will pollute master history once PR is merged, right?

**Comment by gadfort** [comment]
The PR could always be merged by squashing it. But in general it doesn't really matter. @maliberty can chime in if he has anything to add.

**Comment by titan73** [comment]
Ah. I personally find the history important to follow the development of OR and learn from that but it's hard to read due to merge commits (https://github.com/The-OpenROAD-Project/OpenROAD/discussions/5830) and request change commits.

**Comment by maliberty** [comment]
I agree with @gadfort about not force pushing as it disabled the "changes since last review" option in GH.

There are a bunch of constraints that have pushed us towards merge commits but I agree it makes the history harder to follow.  Perhaps we can resolve them and move to squash commits in the future.

**Comment by titan73** [comment]
> I agree with @gadfort about not force pushing as it disabled the "changes since last review" option in GH.

Indeed. No very good.

> 
> There are a bunch of constraints that have pushed us towards merge commits but I agree it makes the history harder to follow. Perhaps we can resolve them and move to squash commits in the future.

No sure automatic squashing is good in all cases. This can be convenient for PR with big changes decomposed into several commits to keep them as is.

Maybe I can instead squash commits and force pushed after approval if the PR becomes a bit messy due to several updates.

**Comment by maliberty** [comment]
If you want a squash commit you can request it and GH allows it as an option at PR merge.  Ideally squash would be the default and non-squash the exception.

**Pull request merged**